cluster/gce/list-resources.sh

```diff
@@ -37,7 +37,17 @@ GREP_REGEX=""
 function gcloud-compute-list() {
   local -r resource=$1
   echo -e "\n\n[ ${resource} ]"
-  gcloud compute ${resource} list --project=${PROJECT} ${@:2} | grep "${GREP_REGEX}"
+  local attempt=1
+  local result=""
+  while true; do
```

Comment:
You'll need a max retires, or a timeout in this loop, otherwise it could go forever.